consensus/src/counters.rs

```diff
@@ -178,6 +178,17 @@ pub static LEADER_REPUTATION_ROUND_HISTORY_SIZE: Lazy<IntGauge> = Lazy::new(|| {
 
 /// Computed at leader election time, with some delay.
 
+/// Current voting power fraction that participated in consensus
+/// (voted or proposed) in the reputation window, used for
```

Comment:
nit: dangling sentence. Also the previous comment (`/// Computed at leader election time, with some delay.`) isn't commenting anything?